docs/view_components/menu_bar.rst

```diff
@@ -38,15 +38,15 @@ Attributes
 Actions
 ^^^^^^^
 
-* Clicking on any of the menus' option results in a drop-down list of sub-menu's options
+* ysdkljasdkasldjasldkad alkd
```

Comment:
Should be ;askdjf;lkadhf;lkadjhfl;ahjdgo;iahjwoivrnavj;cjvk;ajsdf;ojv;oiwjdv

Reply:
```suggestion
;askdjf;lkadhf;lkadjhfl;ahjdgo;iahjwoivrnavj;cjvk;ajsdf;ojv;oiwjdv
```